client/homescreen/layout.js

```diff
@@ -80,7 +73,7 @@ export const Layout = ( {
 			<Fragment>
 				{ showInbox && (
 					<div className="woocommerce-homescreen-column is-inbox">
-						<InboxPanel isPanelEmpty={ isInboxPanelEmpty } />
+						<InboxPanel />
```

Comment:
Can we remove the code that uses `isPanelEmpty` in the `InboxPanel` component as well?

Reply:
@jeffstieler given our conversation of how single column layout will likely remove this functionality, I think that is fair to do. Let me push up a change for that.

Reply:
Actually, since I've based this branch of `main` and the Inbox panel logic has since migrated, that will complicate the picking. Is it okay if we remove that logic in a follow-up PR against `main`?

Reply:
While it is dead/un-used code, the fix here should still be valid is I guess what I'm trying to say but having difficulty doing so 😸